photon/src/main/java/org/pushingpixels/photon/api/transcoder/SvgBatchConverter.java

```diff
@@ -32,96 +32,88 @@
+                    "  outputLanguage=java|kotlin - the language for the transcoded classes",
+                    "  outputFolder=xyz - optional location of output files. If not specified, output files will be placed in the 'sourceFolder'",
+                    "  outputClassNamePrefix=xyz - optional prefix for the class name of each transcoded class"
+            ).forEach(System.out::println);
+            System.out.println(CHECK_DOCUMENTATION);
             System.exit(1);
         }
 
-        String sourceFolderName = getInputArgument(args, "sourceFolder");
-        if (sourceFolderName == null) {
-            System.out.println(
-                    "Missing source folder. Check the documentation for the parameters to pass");
-            System.exit(1);
-        }
-        String outputPackageName = getInputArgument(args, "outputPackageName");
-        if (outputPackageName == null) {
-            System.out.println(
-                    "Missing output package name. Check the documentation for the parameters to " +
-                            "pass");
-            System.exit(1);
-        }
-        String templateFile = getInputArgument(args, "templateFile");
-        if (templateFile == null) {
-            System.out.println(
-                    "Missing template file. Check the documentation for the parameters to pass");
-            System.exit(1);
-        }
-        String outputLanguage = getInputArgument(args, "outputLanguage");
-        if (outputLanguage == null) {
-            System.out.println(
-                    "Missing output language. Check the documentation for the parameters to pass");
-            System.exit(1);
-        }
-        if ((outputLanguage.compareTo("java") != 0) && (outputLanguage.compareTo("kotlin") != 0)) {
-            System.out.println(
-                    "Output language must be either Java or Kotlin. Check the documentation for " +
-                            "the parameters to pass");
-            System.exit(1);
-        }
-        String outputClassNamePrefix = getInputArgument(args, "outputClassNamePrefix");
-        if (outputClassNamePrefix == null) {
-            outputClassNamePrefix = "";
-        }
-        String outputFolderName = getInputArgument(args, "outputFolder");
-        if (outputFolderName == null) {
-            outputFolderName = sourceFolderName;
-        }
+        String sourceFolderName = getInputArgument(args, "sourceFolder", null);
+        Objects.requireNonNull(sourceFolderName, "Missing source folder. " + CHECK_DOCUMENTATION);
 
-        File inputFolder = new File(sourceFolderName);
-        if (!inputFolder.exists()) {
-            return;
-        }
-        File outputFolder = new File(outputFolderName);
-        if (!outputFolder.exists()) {
-            return;
+        String outputPackageName = getInputArgument(args, "outputPackageName", sourceFolderName);
```

Comment:
This is not right. Output package name doesn't default to source folder name (which is a file path that may indeed contain the package name in it).

Reply:
Thanks for spotting this glitch!